Bottom: ef0ebc028a921ab8c8d053eb4bcba1e0f296deba
Top:    d4bee023557090d4db7b227340b2b4cc48835183
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 17:07:59 +0100

Refresh of hid-gtk-add-consolidated-api-f

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 4508add..a78b0d4 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -1354,7 +1354,7 @@ pan_common (GHidPort *port)
   ghid_port_ranges_changed();
 }
 
-static void
+void
 ghid_pan_view_abs (Coord pcb_x, Coord pcb_y, int widget_x, int widget_y)
 {
   gport->view.x0 = SIDE_X (pcb_x) - widget_x * gport->view.coord_per_px;
@@ -1381,7 +1381,7 @@ ghid_pan_view_rel (Coord dx, Coord dy)
  */
 
 #define ALLOW_ZOOM_OUT_BY 10 /* Arbitrary, and same as the lesstif HID */
-static void
+void
 ghid_zoom_view_abs (Coord center_x, Coord center_y, double new_zoom)
 {
   double min_zoom, max_zoom;
@@ -1414,13 +1414,13 @@ ghid_zoom_view_abs (Coord center_x, Coord center_y, double new_zoom)
   ghid_set_status_line_label ();
 }
 
-static void
+void
 ghid_zoom_view_rel (Coord center_x, Coord center_y, double factor)
 {
   ghid_zoom_view_abs (center_x, center_y, gport->view.coord_per_px * factor);
 }
 
-static void
+void
 ghid_zoom_view_fit (void)
 {
   ghid_pan_view_abs (0, 0, 0, 0);
@@ -1428,7 +1428,7 @@ ghid_zoom_view_fit (void)
                                  PCB->MaxHeight / gport->height));
 }
 
-static void
+void
 ghid_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
 {
   int widget_x, widget_y;
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c3e5a54..d42a2de 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1356,7 +1356,7 @@ pan_common (GHidPort *port)
   ghid_port_ranges_changed();
 }
 
-static void
+void
 ghid_pan_view_abs (Coord pcb_x, Coord pcb_y, int widget_x, int widget_y)
 {
   gport->view.x0 = SIDE_X (pcb_x) - widget_x * gport->view.coord_per_px;
@@ -1383,7 +1383,7 @@ ghid_pan_view_rel (Coord dx, Coord dy)
  */
 
 #define ALLOW_ZOOM_OUT_BY 10 /* Arbitrary, and same as the lesstif HID */
-static void
+void
 ghid_zoom_view_abs (Coord center_x, Coord center_y, double new_zoom)
 {
   double min_zoom, max_zoom;
@@ -1416,13 +1416,13 @@ ghid_zoom_view_abs (Coord center_x, Coord center_y, double new_zoom)
   ghid_set_status_line_label ();
 }
 
-static void
+void
 ghid_zoom_view_rel (Coord center_x, Coord center_y, double factor)
 {
   ghid_zoom_view_abs (center_x, center_y, gport->view.coord_per_px * factor);
 }
 
-static void
+void
 ghid_zoom_view_fit (void)
 {
   ghid_pan_view_abs (0, 0, 0, 0);
@@ -1430,7 +1430,7 @@ ghid_zoom_view_fit (void)
                                  PCB->MaxHeight / gport->height));
 }
 
-static void
+void
 ghid_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
 {
   int widget_x, widget_y;
